src/natcap/invest/coastal_vulnerability.py

```diff
@@ -2941,6 +2941,19 @@ def validate(args, limit_to=None):
                 (['shelf_contour_vector_path'],

```

Comment:
It's not related to the SLR issue, but it would be a clear improvement to replace these integer codes with the `ogr` constants like you have them in the SLR test. Do you think it's a good idea to do that while we're here?

Reply:
This comment belongs on the previous line. For some reason github won't let me add a comment to that line.